index.html

```diff
@@ -0,0 +1,206 @@
+            <li><img src="./images/Linkedin.png" alt="linkedin icon"></li>
+            <li><img src="./images/Angelist.png" alt="angel-icon"></li>
+            <li><img src="./images/Twitter.png" alt="Twitter-icon"></li>
+            <li><img src="./images/media.png" alt="m-icon"></li>
+        </ul> 
+        <div class="w-arrow">
+    <ul class="wk-arr">
+        <li><img src="./images/Group55.png" alt="down-arrow icon"></li>
+        <li>work</li>
+    </ul>
+        </div>
+    </section>
+
+</section>
+<main class="main-section">
+    <h2 class="r-work">MY RECENT WORK</h2>
+    <hr class="hr1">
+
+<ul>
+    <li class="bg1">
+
+        <img class="img-p" src="./images/Img-Placeholder.png" alt="img-placeholder">
+        
+        <h3 class="m-stories">Multi-Post Stories</h3>
+        <div>
+        <p class="c-prg">A daily selection of privately personalized reads;
+             no accounts or sign-ups required. has been the industry's
+             standard dummy text ever since the 1500s,
+              when an unknown printer took a standard dummy text.</p>
+        
+        <ul class="c-list">
+            <li>css</li>
+            <li class="c-d">|</li>
+            <li>html</li>
+            <li class="c-d">|</li>
+            <li>Bootstrap</li>
+            <li class="c-d">|</li>
+            <li>ruby</li>
+        </ul>
+        
+        
+        <button type="button" class="btn">See Project</button>
+        </div>
+        
+    </li>
+    <li class="bgi2">
+
+<img class="img-p" src="./images/Img-Placeholder.png" alt="img-placeholder">
+
+<h3 class="m-stories">Multi-Post Stories</h3>
+<div>
+<p class="c-prg">A daily selection of privately personalized reads;
+     no accounts or sign-ups required. has been the industry's
+     standard dummy text ever since the 1500s,
+      when an unknown printer took a standard dummy text.</p>
+
+<ul class="c-list">
+    <li>css</li>
+    <li class="c-d">|</li>
+    <li>html</li>
+    <li class="c-d">|</li>
+    <li>Bootstrap</li>
+    <li class="c-d">|</li>
+    <li>ruby</li>
+</ul>
+
+
+<button type="button" class="btn">See Project</button>
+</div>
+
+    </li>
+    <li class="bg3">
+
+<img class="img-p" src="./images/Img-Placeholder.png" alt="img-placeholder">
+
+<h3 class="m-stories">Multi-Post Stories</h3>
+<div>
+<p class="c-prg">A daily selection of privately personalized reads;
+     no accounts or sign-ups required. has been the industry's
+     standard dummy text ever since the 1500s,
+      when an unknown printer took a standard dummy text.</p>
+
+<ul class="c-list">
+    <li>css</li>
+    <li class="c-d">|</li>
+    <li>html</li>
+    <li class="c-d">|</li>
+    <li>Bootstrap</li>
+    <li class="c-d">|</li>
+    <li>ruby</li>
+</ul>
+
+
+<button type="button" class="btn">See Project</button>
+</div>
+
+    </li>
+    <li class="bg4">
+
+<img class="img-p" src="./images/Img-Placeholder.png" alt="img-placeholder">
+
```

Comment:
- [ ] Image is not showing on your page. This happens for your image src name. You can use `./images/img-Placeholder.png` to fix it.